raaghu-elements/src/rds-alert/rds-alert.tsx

```diff
@@ -64,7 +66,7 @@ const RdsAlert = (props: RdsAlertProps) => {
                         <RdsIcon
                             name={props.icon || " "}
                             fill={props.iconFill}
-                            stroke={props.iconStroke}
+                           // stroke={props.iconStroke}
```

Comment:
@suchithravipin1991 here commented line is a input for that element we cannot comment it out just make iconStroke "false" from alert.stories file and check your changes